uat/custom-components/src/main/java/com/aws/greengrass/artifacts/logGeneratorDiffName.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.artifacts;
+
+import ch.qos.logback.classic.Level;
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.encoder.PatternLayoutEncoder;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.rolling.RollingFileAppender;
+import ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy;
+import ch.qos.logback.core.util.FileSize;
+
+import java.util.function.Consumer;
+
+import static com.aws.greengrass.utils.LocalVerifyHelper.verifyLoggingSucceed;
+import static com.aws.greengrass.utils.LoggingHelper.performLogging;
+import static com.aws.greengrass.utils.UnitConvertHelper.getFileSizeInBytes;
+
+public class logGeneratorDiffName implements Consumer<String[]> {
+    private static final int MAX_HISTORY = 3;
+    private static final String rotationNamePattern = "_%d{yyyy-MM-dd_HH-mm}_%i";
+    private String logFileName;
+    private String logFileExtension;
+    private double fileSize;
+    private String fileSizeUnit;
+    private Long logWriteFreqSeconds;
+    private int logMsgSizeBytes;
+    private int fileSizeBytes;
+    private int totalLogNumbers;
+
+    @Override
+    public void accept(String[] args) {
+        logFileName = args[0];
+        logFileExtension = args[1];
+        fileSize = Double.parseDouble(args[2]);
+        fileSizeUnit = args[3];
+        logWriteFreqSeconds = Long.parseLong(args[4]);
+        logMsgSizeBytes = Integer.parseInt(args[5]);
+        totalLogNumbers = Integer.parseInt(args[6]);
+
+        fileSizeBytes = getFileSizeInBytes(fileSize, fileSizeUnit);
+
+        String currentPath = System.getProperty("user.dir");
+        Logger logger = prepareLogger(currentPath, logFileName, logFileExtension, fileSizeBytes);
+
+        logger.info("hello world !!!!!");
+
+        // starting logging message
+        performLogging(logger, totalLogNumbers, logWriteFreqSeconds, logMsgSizeBytes);
+
+        // print list of log files and results would be presented in local machine, it is easier to verify locally
+        // and manuaal. It is not used for the UAT verification.
+        verifyLoggingSucceed(currentPath, logFileName, logFileExtension);
```

Comment:
I don't think we should add this. If we start printing additional stuff to stdout, the greengrass logger from the nucleus will intercept it and add them to a logging file I believe. Also you could do this by just listing the files on a directory this is getting written to on your machine

Reply:
Actually, this is my purpose. I want to check these info from the logging file that Nucleus writes to, because this file will appear in the /results in my local machine.